Mathlib/Tactic/Linter/FlexibleLinter.lean

```diff
@@ -457,7 +457,12 @@ def flexibleLinter : Linter where run := withSetOptionIn fun _stx => do
       stains := new
 
   for (s, stainStx, d) in msgs do
-    Linter.logLint linter.flexible stainStx m!"'{stainStx}' is a flexible tactic modifying '{d}'…"
+    let (tac, tacOnly) := if stainStx.getKind == ``Lean.Parser.Tactic.simpAll
+      then ("simp_all?", "simp_all only [...]")
+      else ("simp?", "simp only [...]")
```

Comment:
This will silently produce unexpected output if/when we lint on other flexible tactics. What do you think about erroring if the `else` branch is not `simp`? (If that's too high a performance cost, I'm fine with not doing so for now.)

Reply:
We already lint for nonterminal aesop too, so I think it's worth making this change

Reply:
Done!